requirements.txt

```diff
@@ -9,6 +9,7 @@ pytz==2015.7
 requests==2.9.1
 requests-toolbelt==0.6.0
 rfc3987==1.3.4
+strict-rfc3339==0.7
```

Comment:
I don't see this explicitly imported anywhere.
Is this one of those sneaky-conditional python imports for the format checking?